x-pack/plugins/observability_solution/observability_logs_explorer/common/locators/utils/construct_locator_path.ts

```diff
@@ -46,7 +49,14 @@ export const constructLocatorPath = async (params: LocatorPathConstructionParams
       query,
       refreshInterval,
       time: timeRange,
-      columns: columns?.map((field) => ({ field })),
+      columns: columns?.map((column) => {
+        if (column.type === 'smart-field') {
+          return column.name === CONTENT_FIELD
+            ? CONTENT_FIELD_CONFIGURATION
+            : RESOURCE_FIELD_CONFIGURATION;
+        }
+        return column;
+      }),
```

Comment:
**suggestion**: this works for this case, but we could walk an extra mile and write this such that we don't have to update it as soon as we add more smart field configurations:
```ts
// x-pack/plugins/observability_solution/logs_explorer/common/constants.ts
const SMART_FIELD_CONFIGURATIONS = {
  [CONTENT_FIELD]: CONTENT_FIELD_CONFIGURATION,
  [RESOURCE_FIELD]: RESOURCE_FIELD_CONFIGURATION,
};

// x-pack/plugins/observability_solution/observability_logs_explorer/common/locators/utils/construct_locator_path.ts
columns: columns?.map((column) => {
  return column.type === 'smart-field' ? SMART_FIELD_CONFIGURATIONS[column.name] : column;
});
```